pkg/monitor/etcd.go

```diff
@@ -0,0 +1,40 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+func startEtcdMonitoring(ctx context.Context, m Recorder, prometheus prometheusv1.API) {
+	expectedCount := 3
```

Comment:
This expectation assumes no upgrades and no disruptive test scenarios

Reply:
@hexfusion actually, wonder if this could be a function of the pod revision? Is it correct to say after pivot the etcd pods should be at revision 3, and under ideal conditions, a leader election is only expected if we roll out a new revision? If so, would that be a portable value across [non]upgrade jobs?